src/main/java/com/epam/test_generator/dao/impl/JiraStoryDAO.java

```diff
@@ -0,0 +1,150 @@
+package com.epam.test_generator.dao.impl;
+
+import com.epam.test_generator.dao.interfaces.SuitDAO;
+import com.epam.test_generator.entities.Suit;
+import com.epam.test_generator.pojo.JiraStory;
+import java.util.Collections;
+import net.rcarz.jiraclient.*;
+import net.rcarz.jiraclient.Issue.SearchResult;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Component;
+
+import java.lang.reflect.MalformedParametersException;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Component
+public class JiraStoryDAO {
+
+    @Autowired
+    private SuitDAO suitDAO;
+
+    @Autowired
+    private JiraClient client;
+
+
+    private final static String TYPE = "Story";
+    private final static Integer MAX_NUMBER_OF_ISSUES = 10000;
+    private final static Integer CLOSE_ACTION_ID = 31;
+
+    public JiraStory getStoryByJiraKey(String jiraKey) {
+        try {
+            return new JiraStory(client.getIssue(jiraKey));
+        } catch (JiraException e) {
+
+            if (e.getCause() instanceof RestException) {
+                RestException restException = (RestException) e.getCause();
+                if (restException.getHttpStatusCode() == 404) {
+                    return null;
+                }
+            }
+            throw new MalformedParametersException(e.getMessage());
+        }
+    }
+
+
+    /**
+     * Returns list of all stories from Jira that belongs to specific Jira project.
+     * @param jiraProjectKey
+     * @return
+     */
+    public List<JiraStory> getStories(String jiraProjectKey) {
+        List<Issue> issues;
+        try {
+            issues = client.searchIssues("project=" + jiraProjectKey + " AND status in (Open, \"In Progress\", Reopened, Verified) AND type=story ", MAX_NUMBER_OF_ISSUES).issues;
```

Comment:
What if there 10001 issue? Use Integer.MAX_VALUE. And apply codestyle, line is too long.

Reply:
Use String.format to format query string.